mimic/index-dot-html.go

```diff
@@ -0,0 +1,966 @@
+// File generated by 2goarray v0.1.0 (http://github.com/cratonica/2goarray)
+
+package mimic
+
+var indexDotHTML []byte = []byte{
```

Comment:
What is the main reason for embedding this instead of loading it from file and keeping in memory? I'm asking because you took the time to do this, so you probably saw a compelling reason.